runtime/ftplugin/sh.vim

```diff
@@ -44,15 +44,21 @@ if (has("gui_win32") || has("gui_gtk")) && !exists("b:browsefilter")
   let b:undo_ftplugin ..= " | unlet! b:browsefilter"
 endif
 
-if get(b:, "is_bash", 0)
+let s:is_sh = get(b:, "is_bash", 0)
```

Comment:
I suppose this should have been testing for `b:is_sh` instead?

Reply:
Yes, good catch!